### PR TITLE
Add webpack starter

### PR DESCRIPTION
**Pull request by adispezio**
- Add webpack starter
- Remove /react
- Bump typings and remove package lock
- Update README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -195,38 +195,16 @@ This demonstrates:
...
-- bundling plugin code using Webpack, and
-- using Vue Single File Components.
-
-[Check out the source code.](vue/)
+    $ npm install
+    $ npm run build
 
+[Check out the source code.](webpack-react/)
 
-[docs]: https://www.figma.com/plugin-docs/figma-figjam-plugins/
```
Comment (brsbl) [inline README.md:225]: uh oh, unintentional?

File: README.md [hunk 2]
```diff
@@ -195,38 +195,16 @@ This demonstrates:
```
Comment (brsbl) [inline README.md:195]: let's update the header to be `Webpack & React`?

**Review by brsbl — CHANGES_REQUESTED** [review]
smol requested changes~

File: README.md (hunk 1 above)
Comment (adispezio) [reply README.md:225]: Woops, added back in!

File: README.md (hunk 2 above)
Comment (adispezio) [reply README.md:195]: Updated! We may want to revisit ampersands vs plus signs later.

Comment (brsbl) [reply README.md:195]: ooo yeah plus sign is fine. im indifferent

**Review by brsbl — APPROVED** [review]

**Pull request merged**